public/js/monitor.js

```diff
@@ -312,6 +312,19 @@ function resizeDashboardMargin() {
   }
 }
 
+function checkIfTier1(preferredLanguage) {
+  const tier1Languages = ["de", "en", "fr"];
+  let isTier1Language = false;
+
+  tier1Languages.forEach( lang => {
+    if (preferredLanguage.includes(lang) ) {
+      isTier1Language = true;
+    }
+  });
+
+  return isTier1Language;
```

Comment:
This seems to work too, right?
```suggestion
  const tier1Languages = ["de", "en", "fr"];

  tier1Languages.forEach( lang => {
    if (preferredLanguage.includes(lang) ) {
      return true;
    }
  });

  return false;
```

Reply:
Sure does. Accepted!

Reply:
I... I dont think that's how it works. I think your nested `return true` doesnt return the _outer_ function it returns the nested forEach arrow function, so you'll probably always `return false` here.

```js
console.log("ru:", test("ru"));
console.log("de:", test("de"));
console.log("en:", test("en"));
console.log("fr:", test("fr"));

function test(preferredLanguage="") {
  const tier1Languages = ["de", "en", "fr"];
  tier1Languages.forEach( lang => {
    if (preferredLanguage.includes(lang) ) {
      return true;
    }
  });
  return false;
}

/*
ru: false
de: false
en: false
fr: false
*/
```